### ISSUE TITLE
"setDashboardTime" disables refresh interval

### ISSUE THREAD

**Issue by MaximBG** [Webiks/kibana-API#47]
"setDashboardTime" was the last "case" inside switch statement. When you added "setRefreshInterval" after it you forgot to put "break" statement and execution flow continues to "setRefreshInterval" and refresh interval affected.
Another issue that reproduces itself about 50% of the time: refresh interval does not appear in globalState when you build URL for Kibana and therefore it being reset to 0.

**Comment by ytzlax** [comment]
Hi,
I don't understand the second part of the issue, can you explain?

**Comment by MaximBG** [comment]
The issue was in 6.3 and we switched to version 6.4 since in 6.3 it didn't work as expected. In 6.4.2 it works perfect.
With regard to the issues in 6.3.
1. Setting time interval resets refresh to 0.
2. Setting refresh time works 50% of time. Once works, once no. Pattern is: 1010101...
3. Similar issue with set interval but pattern is 1001001001... (1 means that operation succeded and 0 the time interval/refresh didn't change)
We tried to implement workarounds but finally decided to switch to the version where everything works.
